### PR TITLE
test: try stabilize test case `test_get_lock_wait_info_api`

### LINKED ISSUE #10450: Unstable test `server::kv_service::test_get_lock_wait_info_api`
## Bug Report
```
[2021-06-23T16:50:18.634Z] test server::kv_service::test_get_lock_wait_info_api ... thread 'main' panicked at 'called `Option::unwrap()` on a `None` value', tests/integrations/server/kv_service.rs:1717:27

[2021-06-23T16:50:18.634Z] stack backtrace:

[2021-06-23T16:50:18.634Z]    0: rust_begin_unwind

[2021-06-23T16:50:18.634Z]              at /rustc/16bf626a31cb5b121d0bca2baa969b4f67eb0dab/library/std/src/panicking.rs:493:5

[2021-06-23T16:50:18.634Z]    1: core::panicking::panic_fmt

[2021-06-23T16:50:18.634Z]              at /rustc/16bf626a31cb5b121d0bca2baa969b4f67eb0dab/library/core/src/panicking.rs:92:14

[2021-06-23T16:50:18.634Z]    2: core::panicking::panic

[2021-06-23T16:50:18.634Z]              at /rustc/16bf626a31cb5b121d0bca2baa969b4f67eb0dab/library/core/src/panicking.rs:50:5

[2021-06-23T16:50:18.634Z]    3: core::option::Option<T>::unwrap

[2021-06-23T16:50:18.634Z]              at /rustc/16bf626a31cb5b121d0bca2baa969b4f67eb0dab/library/core/src/option.rs:386:21

[2021-06-23T16:50:18.634Z]    4: integrations::server::kv_service::test_get_lock_wait_info_api

[2021-06-23T16:50:18.634Z]              at /home/jenkins/tikv-src/tests/integrations/server/kv_service.rs:1717:19

[2021-06-23T16:50:18.634Z]    5: integrations::server::kv_service::test_get_lock_wait_info_api::{{closure}}

[2021-06-23T16:50:18.634Z]              at /home/jenkins/tikv-src/tests/integrations/server/kv_service.rs:1692:1

[2021-06-23T16:50:18.634Z]    6: core::ops::function::FnOnce::call_once

[2021-06-23T16:50:18.634Z]              at /rustc/16bf626a31cb5b121d0bca2baa969b4f67eb0dab/library/core/src/ops/function.rs:227:5

[2021-06-23T16:50:18.634Z]    7: test_util::runner::run_test_with_hook::{{closure}}::{{closure}}

[2021-06-23T16:50:18.634Z]              at /home/jenkins/tikv-src/components/test_util/src/runner.rs:64:21

[2021-06-23T16:50:18.634Z]    8: core::ops::function::FnOnce::call_once{{vtable.shim}}

[2021-06-23T16:50:18.634Z]              at /rustc/16bf626a31cb5b121d0bca2baa969b4f67eb0dab/library/core/src/ops/function.rs:227:5

[2021-06-23T16:50:18.634Z]    9: <alloc::boxed::Box<F,A> as core::ops::function::FnOnce<Args>>::call_once

[2021-06-23T16:50:18.634Z]              at /rustc/16bf626a31cb5b121d0bca2baa969b4f67eb0dab/library/alloc/src/boxed.rs:1546:9

[2021-06-23T16:50:18.634Z] note: Some details are omitted, run with `RUST_BACKTRACE=full` for a verbose backtrace.

[2021-06-23T16:50:18.634Z] thread '<unnamed>' panicked at 'called `Result::unwrap()` on an `Err` value: RpcFailure(RpcStatus { code: 14-UNAVAILABLE, message: "Socket closed", details: [] })', components/test_raftstore/src/util.rs:970:38

[2021-06-23T16:50:18.634Z] stack backtrace:

[2021-06-23T16:50:18.634Z]    0: rust_begin_unwind

[2021-06-23T16:50:18.634Z]              at /rustc/16bf626a31cb5b121d0bca2baa969b4f67eb0dab/library/std/src/panicking.rs:493:5

[2021-06-23T16:50:18.634Z]    1: core::panicking::panic_fmt

[2021-06-23T16:50:18.634Z]              at /rustc/16bf626a31cb5b121d0bca2baa969b4f67eb0dab/library/core/src/panicking.rs:92:14

[2021-06-23T16:50:18.634Z]    2: core::result::unwrap_failed

[2021-06-23T16:50:18.635Z]              at /rustc/16bf626a31cb5b121d0bca2baa969b4f67eb0dab/library/core/src/result.rs:1355:5

[2021-06-23T16:50:18.635Z]    3: core::result::Result<T,E>::unwrap

[2021-06-23T16:50:18.635Z]              at /rustc/16bf626a31cb5b121d0bca2baa969b4f67eb0dab/library/core/src/result.rs:1037:23

[2021-06-23T16:50:18.635Z]    4: test_raftstore::util::kv_pessimistic_lock_with_ttl

[2021-06-23T16:50:18.635Z]              at /home/jenkins/tikv-src/components/test_raftstore/src/util.rs:970:5

[2021-06-23T16:50:18.635Z]    5: integrations::server::kv_service::test_get_lock_wait_info_api::{{closure}}

[2021-06-23T16:50:18.635Z]              at /home/jenkins/tikv-src/tests/integrations/server/kv_service.rs:1702:9

[2021-06-23T16:50:18.635Z] note: Some details are omitted, run with `RUST_BACKTRACE=full` for a verbose backtrace.

[2021-06-23T16:50:18.635Z] FAILED
```

### What version of TiKV are you using?

master

### Steps to reproduce

in ci https://ci.pingcap.net/blue/organizations/jenkins/tikv_ghpr_test/detail/tikv_ghpr_test/2999/pipeline

### PR DESCRIPTION
**Pull request by longfangsong**
Signed-off-by: longfangsong <longfangsong@icloud.com>

### What problem does this PR solve?

Issue Number: close #10450

Problem Summary:

`server::kv_service::test_get_lock_wait_info_api` is unstable.

### What is changed and how it works?

Use `kv_pessimistic_lock_with_ttl` instead of `must_kv_pessimistic_lock` to give the lock a larger ttl for the first lock acquired.
So the old lock won't exceed its ttl when doing the test process, which IMO cause the test unstable.

### Check List 

Tests 

- Integration test

### Release note 

```release-note
None.
```

### CODE REVIEW

File: tests/integrations/server/kv_service.rs [hunk 1]
```diff
@@ -1705,7 +1705,7 @@ fn test_get_lock_wait_info_api() {
 
     let mut ctx1 = ctx.clone();
     ctx1.set_resource_group_tag(b"resource_group_tag1".to_vec());
-    must_kv_pessimistic_lock(&client, ctx1, b"a".to_vec(), 20);
+    kv_pessimistic_lock_with_ttl(&client, ctx1, vec![b"a".to_vec()], 20, 20, false, 1000);
```
Comment (youjiali1995) [inline tests/integrations/server/kv_service.rs:1708]: I don't think it takes effect. The lock shouldn't be resolved.

Comment (MyonKeminta) [reply tests/integrations/server/kv_service.rs]: I think this makes sense. The previously reported failure is possible to be caused by the lock (which is the blocker) expires before querying the get_lock_wait_info api.

**Review by MyonKeminta — APPROVED** [review]

**Review by zhouqiang-cl — APPROVED** [review]
LGTM

**Pull request merged**